src/conditionals/programs_with_decision_points/hard/q14/checkWord.java

```diff
@@ -0,0 +1,58 @@
+package conditionals.programs_with_decision_points.hard.q14;
+/*
+Write a program that asks the user for two words, word1 and word2.
+Then, check if the combined length of both words:
+Less than 10: Print "Short phrase."
+Between 10 and 15 (inclusive): Print "Medium phrase."
+Greater than 15: If word1 starts with lowercase "a", print "a-start long phrase."
+Otherwise, print "Long phrase."
+ */
+import java.util.Scanner;
+
+public class checkWord {
+
+    public static void main(String[] args) {
+        Scanner scanner = new Scanner(System.in);
+
+        System.out.println("Enter word1 ");
```

Comment:
Same as last pr, ad colons for site-wide consistency (also why I've been reformatting the questions)